### ISSUE TITLE
210603 인공지능 오피스 아워 이슈

### ISSUE THREAD

**Issue by makeitmin** [mirror-look/mirror-look-origin#68]
In GitLab by @Jaemin on Jun 3, 2021, 19:02

## 개발 시 어려운 점을 아래 리스트 중에 체크해주세요.
| 이슈 Label| 해당 사항 체크(✅,❎) |
| ------ | ------ |
| 전체적인 아이디어, 프로젝트 진행 방향성에 관한 이슈 |  |
| To-do 기능 구현의 어려움 | ✅ |
| 딥러닝 결과 해석의 어려움 |  |
| 버그 |  |
| 서버 이슈 |  |
| 속도 이슈 |  |
| gitlab 이슈 |  |
| 그 외 |  |

- 이슈에 대한 자세한 설명 :

### 1. 개발 시 이슈가 무엇이었는지 예상 동작과 현재 동작에 대해 자세히 기입해주세요.

https://kdt-gitlab.elice.io/001-part4-aifashionimage/team1/mirror-look/-/blob/feature/AI/AI/Final/Final.ipynb

이전에 학습했던 모델의 데이터 셋과 동일한 데이터 셋을 단지 카테고리 분류를 다르게 했기 때문에,
전에 썼던 모델을 똑같이 사용해서 학습해보았는데 학습이 잘 되지 않는 것 같습니다.

### 2. 이슈에 대한 원인을 기입해주세요.

원인을 아직 잘 모르겠지만... learning rate를 조금 느리게 주는 것은 어떨지 궁금합니다.

### 3. 문제가 발생한 코드 및 스크린샷을 기입해주세요.

![image](https://my-gitlab-bucket.s3.amazonaws.com/377863191/46b96cb2363e371cd0ada23ac154b9d7.png)
![image](https://my-gitlab-bucket.s3.amazonaws.com/377863191/5f3dd55a33fcd306f7606840215de065.png)
![image](https://my-gitlab-bucket.s3.amazonaws.com/377863191/61681c6cfa28970b7e6034ae27749a61.png)
![image](https://my-gitlab-bucket.s3.amazonaws.com/377863191/ccd4707348fb10e3ba7e9d7a9e26fac7.png)
![image](https://my-gitlab-bucket.s3.amazonaws.com/377863191/f5ab38eec7b5e83debed40df6abdcc19.png)
![image](https://my-gitlab-bucket.s3.amazonaws.com/377863191/e8ebb0943a5d198750cec6a65268a392.png)

### 4. 이슈를 어떻게 고칠 것인지 자세히 설명해주세요. (해결 방법을 못 찾을 시, 5. 궁금한 점에 기입해주세요)

- 코치님께 피드백을 받고 수정한다.

**Comment by makeitmin** [comment]
In GitLab by @Jaemin on Jun 3, 2021, 19:02

changed due date to June 03, 2021

**Comment by makeitmin** [comment]
In GitLab by @hyoje on Jun 4, 2021, 20:29

overfitting 해결을 위해서 
1) resnet모델을 freeze
2) batch normalization 사용
3) 학습 파라미터 줄이기

등 을 시도해보시면 좋을 것 같습니다
